pkg/api-server/server.go

```diff
@@ -99,9 +105,18 @@ func NewApiServer(resManager manager.ResourceManager, clusters poller.ZoneStatus
 	container.Add(clustersWs)
 
 	container.Filter(cors.Filter)
-	return &ApiServer{
-		server: srv,
-	}, nil
+
+	newApiServer := &ApiServer{
+		server:          srv,
+		GuiServerConfig: cfg.GuiServer,
+	}
+
+	// Handle the GUI
+	container.Handle("/gui/", http.StripPrefix("/gui/", http.FileServer(resources.GuiDir)))
+	container.Handle("/api/", http.StripPrefix("/api/", container))
+	container.ServeMux.HandleFunc("/gui/config/", newApiServer.configHandler)
```

Comment:
I don't think `/api` is needed anymore. You can just invoke root path of this server. I also think that `/gui/config` is obsolete, because all the information are in `/config`

Can you coordinate this with @bloqhead on GUI side? I can help if needed.

Reply:
I also suggest API on `/`, while the GUI on `/gui`.

Reply:
I'm not opposed to using `/` for the API. I have to traverse up the URL path to access the API if the GUI is in `/gui` but that change is small. I think labeling them `/api` and `/gui` adds more clarity, but it's not crucial.

Reply:
👍